cpp/msbuild/ice.nuget.targets

```diff
@@ -14,36 +14,36 @@
                                   $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\icedb*.lib;
                                   $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\icestormservice*.lib;
                                   $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\iceutil*.lib;
-                                  $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\icexml*.lib;
                                   $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\slice*.lib"/>
     </ItemGroup>
 
-    <!-- Executables & Tools -->
-    <ItemGroup>
-
-        <ExecutableExcludes Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\glacier2router.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceboxadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icebridge.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegridadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegriddb.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceserviceinstall.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormdb.exe"
-                            Condition="'$(DefaultBuild)' != '$(Platform)|$(Configuration)' and '$(Configuration)' == 'Debug'"/>
-
-        <ExecutableExcludes Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\slice2*.exe"/>
-
-        <Executables Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\*.exe"
-                     Exclude="@(ExecutableExcludes)"/>
+    <ItemGroup Condition="'$(DefaultBuild)' != '$(Platform)|$(Configuration)' and '$(Configuration)' == 'Debug'">
```

Comment:
I am guessing we do that to reduce the size of the NuGet?

However, this is also a pain as you need need to have multiple bin\$(Platform)\$(Configuration) in your PATH *in the right order* unless you use the default platform with Release.

It's also odd to "exclude" some services (dsnode, glacier2router) but keep others (icegridnode, icegridregistry, IceStorm).

Reply:
> I am guessing we do that to reduce the size of the NuGet?

It seems so but I don't remember, this is the logic we have in 3.7.